README.md

```diff
@@ -26,138 +23,131 @@ Kubernetes is:
-  - **User documentation**
-    - for people who want to run programs on an existing Kubernetes cluster
-    - in the [Kubernetes User Guide: Managing Applications](http://kubernetes.github.io/docs/user-guide/)
-	*Tip: You can also view help documentation out on [http://kubernetes.io/docs/](http://kubernetes.io/docs/).*
-    - the [Kubectl Command Line Interface](http://kubernetes.io/docs/user-guide/kubectl-overview/) is a detailed reference on
-      the `kubectl` CLI
-    - [User FAQ](https://github.com/kubernetes/kubernetes/wiki/User-FAQ)
-  - **Cluster administrator documentation**
-    - for people who want to create a Kubernetes cluster and administer it
-    - in the [Kubernetes Cluster Admin Guide](http://kubernetes.io/docs/admin/)
-  - **Developer and API documentation**
-    - for people who want to write programs that access the Kubernetes API, write plugins
-      or extensions, or modify the core Kubernetes code
-    - in the [Kubernetes Developer Guide](docs/devel/README.md)
-    - see also [notes on the API](docs/api.md)
-    - see also the [API object documentation](docs/api-reference/README.md), a
-      detailed description of all fields found in the core API objects
-  - **Walkthroughs and examples**
-    - hands-on introduction and example config files
-    - in the [user guide](http://kubernetes.github.io/docs/user-guide/)
-    - in the [docs/examples directory](examples/)
-  - **Contributions from the Kubernetes community**
-    - in the [docs/contrib directory](contrib/)
-  - **Design documentation and design proposals**
-    - for people who want to understand the design of Kubernetes, and feature proposals
-    - design docs in the [Kubernetes Design Overview](docs/design/README.md) and the [docs/design directory](docs/design/)
-    - proposals in the [docs/proposals directory](docs/proposals/)
-  - **Wiki/FAQ**
-    - in the [wiki](https://github.com/kubernetes/kubernetes/wiki)
-    - troubleshooting information in the [troubleshooting guide](http://kubernetes.io/docs/troubleshooting/)
-
-## Community, discussion, contribution, and support
-
-See which companies are committed to driving quality in Kubernetes on our [community page](http://kubernetes.io/community/).
-
-Do you want to help "shape the evolution of technologies that are container packaged, dynamically scheduled and microservices oriented?"
-
-You should consider joining the [Cloud Native Computing Foundation](https://cncf.io/about). For details about who's involved and how Kubernetes plays a role, read [their announcement](https://cncf.io/news/announcement/2015/07/new-cloud-native-computing-foundation-drive-alignment-among-container).
-
-### Code of conduct
-
-Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
-
-### Are you ready to add to the discussion?
-
-We have presence on:
+Join us on social media and read our blog:
```

Comment:
I'd prefer to link to kubernetes.io (e.g., http://kubernetes.io/community/) and keep this simpler. If some of this info isn't easily found there, we should fix that.